src/pkg/packaging/deb/package.targets

```diff
@@ -165,38 +223,80 @@
     <MakeDir Directories="$(debLayoutPackageRoot)" />
     <MakeDir Directories="$(debLayoutSamples)" />
     <MakeDir Directories="$(debLayoutDocs)" />
-    
+
     <!-- Copy files to debian layout -->
     <ItemGroup>
-        <SHFiles Include="$(InputRoot)/**/*" />
-        <SHManpages Include="$(ManPagesDir)/**/*" />
+      <SHFiles Include="$(InputRoot)/**/*" />
+      <SHManpages Include="$(ManPagesDir)/**/*" />
     </ItemGroup>
 
     <Copy SourceFiles="@(SHFiles)" DestinationFolder="$(debLayoutPackageRoot)"/>
     <Copy SourceFiles="@(SHManpages)" DestinationFolder="$(debLayoutDocs)"/>
 
     <!-- Replace config json variables -->
     <ItemGroup>
-      <ConfigPattern Include="%25SHARED_HOST_DEBIAN_VERSION%25" /> <ConfigReplace Include="$(HostVersion)" />
```

Comment:
@chcosta, I don't see tokens %SHARED_HOST_DEBIAN_VERSION% and %HOSTFXR_DEBIAN_VERSION% in ...packaging/deb/dotnet-sharedframework-debian_config.json. 
Can you check if these are really required now?